OCP-Infrastructure/helm/hogarama/charts/amq/files/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+/opt/amq/bin/launch.sh nostart
```

Comment:
Duplicate file?

Reply:
Duplicate of?

Reply:
OCP-Infrastructure/configs/amq/entrypoint.sh

Reply:
Then, no duplicate as such. One is for the bash scripted version, one is for the helm version. Bash scripted version will be removed in a future merge request once helm is established

Reply:
Thats exactly my point further down. If the helm version works for this component then there is no need to include the bash-version. If the helm-version doesn't work, there is no need to include it in this merge request. If you can't migrate bash to helm component by component independently, then there is something wrong with the setup (which is probably the case here). 
If people should work with this you can't expect them to edit all these duplicates like three broker.xml